From baad30421525ae94e5bf609643436e6b32030a04 Mon Sep 17 00:00:00 2001
From: Feng, Boqun <boqun.feng@intel.com>
Date: Mon, 20 Jun 2011 06:04:33 +0000
Subject: i965_drv_video: h264 shader kernels makefile fix

_export.inc is a temporary file in kernels compile stage, need to
create one before preprocess and compile, and it will be deleted
after that.
---
diff --git a/i965_drv_video/shaders/h264/mc/Makefile.am b/i965_drv_video/shaders/h264/mc/Makefile.am
index 3c80c5d..158f14c 100644
--- a/i965_drv_video/shaders/h264/mc/Makefile.am
+++ b/i965_drv_video/shaders/h264/mc/Makefile.am
@@ -167,6 +167,7 @@ SUFFIXES = .g4a .g4b
 	m4 $*.g4a > $*.g4m && intel-gen4asm -o $@ $*.g4m && intel-gen4asm -g 5 -o $@.gen5 $*.g4m && rm $*.g4m
 
 $(INTEL_MC_G4B): $(INTEL_MC_ASM) $(INTEL_ILDB_ASM)
+	touch _export.inc
 	@cpp -D DEV_CTG -I ../ildb/ AllAVC.asm > a.asm; \
 	../../gpp.py a.asm avc_mc.asm; \
 	intel-gen4asm -l list -a -e _export.inc -o $@ -g 4 avc_mc.asm; \
@@ -177,6 +178,7 @@ $(INTEL_MC_G4B): $(INTEL_MC_ASM) $(INTEL_ILDB_ASM)
 	rm a.asm avc_mc.asm _export.inc
 
 $(INTEL_MC_G4B_GEN5): $(INTEL_MC_ASM) $(INTEL_ILDB_ASM)
+	touch _export.inc
 	@cpp -D DEV_ILK -I ../ildb/ AllAVC.asm > a.asm; \
 	../../gpp.py a.asm avc_mc.asm; \
 	intel-gen4asm -l list -a -e _export.inc -o $@ -g 5 avc_mc.asm; \
--
cgit v0.8.3-6-g21f6
